modal_proto/api.proto

```diff
@@ -1550,6 +1550,17 @@ message MultiPartUpload {
   string completion_url = 3;
 }
 
+message NetworkAccess {
+  enum NetworkAccessType {
+    UNSPECIFIED = 0;
+    OPEN = 1;
+    BLOCKED = 2;
+    ALLOWLIST = 3;
```

Comment:
Why not `DENYLIST` / `ALLOWLIST` or `BLOCKED` / `ALLOWED`?

Reply:
I don't want to support a denylist - I think allowlists are more robust. As for `ALLOWED`, it felt a little ambiguous to me since technicaly `ALLOWLIST` is allowed as well - `OPEN` indicates there are no restrictions.